lib/ansible/modules/windows/win_nssm.py

```diff
@@ -94,6 +94,10 @@
       - auto
       - manual
       - disabled
+  environment:
+    version_added: "2.5"
+    description:
+      - Add extra environemnt variables to the service's default environment
```

Comment:
Needs a full stop at the end. Would also be good to see an example of how to use this.